src/ui/report.ui

```diff
@@ -0,0 +1,721 @@
+                    <child>
+                      <object class="GtkLabel">
+                        <property name="halign">start</property>
+                        <property name="label">Reporter email address</property>
+                        <property name="margin-bottom">10</property>
+                        <style>
+                          <class name="normal-text"/>
+                        </style>
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkEntry" id="a_7">
+                        <property name="hexpand">True</property>
+                        <property name="placeholder-text">Please enter an email</property>
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkLabel" id="required_7">
+                        <property name="halign">start</property>
+                        <property name="label">The question is required.</property>
+                        <property name="margin-start">5</property>
+                        <property name="visible">False</property>
+                        <style>
+                          <class name="required-text"/>
+                        </style>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+              </object>
+            </child>
+          </object>
+        </property>
+        <property name="vexpand">True</property>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox" id="footer">
+        <property name="orientation">vertical</property>
+        <property name="valign">end</property>
+        <property name="width-request">0</property>
+        <child>
+          <object class="GtkSeparator"/>
+        </child>
+        <child>
+          <object class="GtkBox">
+            <property name="orientation">horizontal</property>
+            <child>
+              <object class="GtkBox">
+                <property name="halign">start</property>
+                <property name="width-request">60</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkBox">
+                <property name="hexpand">True</property>
+                <child>
+                  <object class="GtkLabel" id="summary">
+                    <property name="hexpand">True</property>
+                    <property name="visible">False</property>
+                    <style>
+                      <class name="success-text"/>
+                    </style>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="GtkBox">
+                <property name="halign">end</property>
+                <child>
+                  <object class="GtkButton" id="submit">
```

Comment:
There should be space between the button and box

Reply:
The parent box is removed